assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,30 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
+let myFunction = () => {};
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
+let anotherFunction = (param) => param;  // don't need brackets for just one param
```

Comment:
don't need `()` for one param. The `{}` is for when you have more logic going on in your function than just the return. Without the brackets, the right side of the `=>` is returned. This is called implicit return. With the brackets, you need to have a return statement.